lib/sendserver_async.c

```diff
@@ -0,0 +1,1050 @@
+/*
+ * Copyright (C) 1995,1996,1997 Lars Fenneberg
+ *
+ * Copyright 1992 Livingston Enterprises, Inc.
+ *
+ * Copyright 1992,1993, 1994,1995 The Regents of the University of Michigan
+ * and Merit Network, Inc. All Rights Reserved
+ *
+ * Copyright (C) 2022 Cadami GmbH, info@cadami.net
```

Comment:
This seem to be on a strange position. I'd expect it to be on the top together with the BSD2 boilerplate.